atlasdb-commons/src/main/java/com/palantir/common/base/BatchingVisitableView.java

```diff
@@ -225,7 +225,7 @@ public T getMin(Ordering<? super T> ordering, @Nullable T defaultElement) {
     public ImmutableList<T> immutableCopy() {
         final ImmutableList.Builder<T> builder = ImmutableList.builder();
         delegate().batchAccept(
-                BatchingVisitables.KEEP_ALL_BATCH_SIZE,
+                BatchingVisitables.DEFAULT_BATCH_SIZE,
```

Comment:
is there a way to make this use the batch hint specified in the range request? from my experience that is where people set their batch size

Reply:
I don't see any visibility to the original range request here. Take a look at @jnelson15's latest comment, and the release note, for a possible workaround.

Happy to discuss this offline - it feels like there should be some way to do this.